lib/stoplight/notifier/slack.rb

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+module Stoplight
+  module Notifier
+    # @see Base
+    class Slack < Base
+      # @return [Proc]
+      attr_reader :formatter
+
+      # @return [::Slack::Notifier]
+      attr_reader :slack_notifier
+
+      # @param slack_notifier [::Slack::Notifier]
+      # @param formatter [Proc, nil]
+      def initialize(slack_notifier, formatter = nil)
```

Comment:
Does this need to accept any options? I see that slack-notifier has a default [channel and username](https://github.com/stevenosloan/slack-notifier/blob/1.2.1/readme.md#setting-defaults).